sci-biology/sailfish/sailfish-0.9.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
as of today, `portage-2.2.25` is removed from prefix again, so I've got no way of checking `EAPI=6` ebuilds with `repoman`